source/code/plugins/filter_scom_excl_correlation.rb

```diff
@@ -0,0 +1,73 @@
+module Fluent
```

Comment:
I will like to open a new PR to extract out common code in the filter plugins after merging this PR. This will give us a base agent for our functionality which we can use to improve further.